creator-node/src/routes/tracks.js

```diff
@@ -772,6 +772,47 @@ router.get(
       )
     }
 
+    // attempt to fetch cached stream data for blockchainId
+    // if found, bypass the db and discovery queries and proceed to getCID
+    try {
+      const cachedData = await redisClient.get(
```

Comment:
concern about fetching entire track file into memory? with current implementation we stream it directly to client

Reply:
the only thing cached is CID so it only fetches a single CID. track itself is not cached

Reply:
silly me - i should know better than to leave comments before full review haha